resources/servo.css

```diff
@@ -7,6 +7,7 @@ input[type="reset"]     { background: lightgrey; border-top: solid 1px #EEEEEE;
 input[type="hidden"]    { display: none !important }
 input[type="checkbox"],
 input[type="radio"]     { font-family: monospace !important; border: none !important; background: transparent; }
+input[dir="rtl"]        { text-align: right; }
```

Comment:
That's probably because our code that determines the behaviour of `dir="auto"` isn't working right. I think we should merge these changes and file a followup issue to fix that.